src/ImGui/Desktop/Fusee.ImGui.Desktop/Templates/ImGuiFolderPicker.cs

```diff
@@ -0,0 +1,372 @@
+                    var name = fse.Name;
+
+                    ImGui.PushStyleColor(ImGuiCol.Text, FolderColor.ToUintColor());
+                    ImGui.PushStyleColor(ImGuiCol.Header, SelectedColor.ToUintColor());
+
+                    if (ImGui.Selectable(name + "/", CurrentlySelectedFolder?.Name == name, ImGuiSelectableFlags.DontClosePopups | ImGuiSelectableFlags.AllowDoubleClick))
+                    {
+                        CurrentlySelectedFolder = fse;
+                        if (ImGui.IsMouseDoubleClicked(0))
+                        {
+                            if (ImGui.GetIO().WantCaptureMouse)
+                            {
+                                LastOpenendFolders.Push(CurrentOpenFolder);
+                                CurrentOpenFolder = fse;
+                            }
+                        }
+                    }
+
+                    ImGui.PopStyleColor();
+                    ImGui.PopStyleColor();
+                }
+
+
+                ImGui.PopStyleColor();
+                ImGui.PopStyleVar();
+                ImGui.EndChild();
+            }
+
+            //// Folder Selector
+            ImGui.NewLine();
+            ImGui.BeginChild($"FileSelector##{_folderPickerCount}", new Vector2(-1, -1), false, ImGuiWindowFlags.AlwaysAutoResize);
+
+            var selectedFile = CurrentlySelectedFolder?.Name;
+            ImGui.SetNextItemWidth(FileTextInputWidth - ImGui.CalcTextSize(FileLabelTxt).X - ImGui.GetStyle().ItemSpacing.X);
+            ImGui.Dummy(new Vector2(-1, -1));
+
+            var sameLineOffset = WinSize.X - WindowPadding.X - (BottomButtonSize.X * 2 + ImGui.GetStyle().ItemSpacing.X * 4);
+
+            if (CurrentlySelectedFolder != null && CurrentlySelectedFolder.Exists)
+            {
+                ImGui.SameLine(sameLineOffset);
+                if (ImGui.Button($"{PickedFileTxt}##{_folderPickerCount}", BottomButtonSize))
+                {
+                    if (CurrentlySelectedFolder != null)
+                        OnPicked?.Invoke(this, CurrentlySelectedFolder);
+                    else
+                        OnPicked?.Invoke(this, CurrentOpenFolder);
+                    filePickerOpen = false;
+                }
+            }
+            else
+            {
+                ImGui.SameLine(sameLineOffset);
+                ImGui.BeginDisabled();
+                ImGui.Button(PickedFileTxt, BottomButtonSize);
+                ImGui.EndDisabled();
+            }
+
+            ImGui.SameLine();
+            if (ImGui.Button($"{CancelFileOpenTxt}##{_folderPickerCount}", BottomButtonSize))
+            {
+                OnCancel?.Invoke(this, EventArgs.Empty);
+                filePickerOpen = false;
+            }
+
+            ImGui.EndChild();
+
+            ImGui.End();
+
+            ImGui.PopStyleVar(2);
+            ImGui.PopStyleColor();
+
+            return;
+        }
+
+        private static List<DirectoryInfo> GetFileSystemEntries(string fullName)
```

Comment:
See comment for `GetFileSystemEntries` in `ImGuiFilePicker`.

Reply:
Changed accordingly.